openvpncloud/resource_user_test.go

```diff
@@ -0,0 +1,118 @@
+package openvpncloud
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/OpenVPN/terraform-provider-openvpn-cloud/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccOpenvpncloudUser_basic(t *testing.T) {
+	rn := "openvpncloud_user.test"
+	user := client.User{
+		Username:  acctest.RandStringFromCharSet(10, alphabet),
+		FirstName: acctest.RandStringFromCharSet(10, alphabet),
+		LastName:  acctest.RandStringFromCharSet(10, alphabet),
+		Email:     "v.kozyrev.sa@gmail.com", // replace it with a test email later
```

Comment:
please, replace this before merging to main

Reply:
fixed